src/telliot_feeds/reporters/tellor_360.py

```diff
@@ -198,7 +199,13 @@ async def rewards(self) -> int:
         total_rewards: int = 0
 
         if self.datafeed is not None:
-            fetch_autopay_tip = await fetch_feed_tip(self.autopay, self.datafeed.query.query_id)
+            try:
+                qid = self.datafeed.query.query_id
+                fetch_autopay_tip = await fetch_feed_tip(self.autopay, qid)
+            except EncodingTypeError:
+                logger.warning("Unable to generate query data/id for datafeed")
+                fetch_autopay_tip = 0
```

Comment:
Is this needed, since its already set on 198?

Reply:
Good catch, I removed it